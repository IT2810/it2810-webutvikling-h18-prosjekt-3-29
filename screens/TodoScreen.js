import React from 'react';
import {
  ScrollView,
  StyleSheet,
  View,
} from 'react-native';
import TodosContainer from '../components/TodosContainer';

export default class TodoScreen extends React.Component {
  static navigationOptions = {
    title: 'Todo',
  };

  render() {
    return (
      <View style={styles.container}>
        <ScrollView style={styles.container} contentContainerStyle={styles.contentContainer}>
          <View style={styles.todoContainer}>
            <TodosContainer addFinishedTodosToAsync={this.addFinishedTodosToAsync}/>
          </View>
        </ScrollView>
      </View>
    );
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fcfcfc',
  },
  contentContainer: {
    paddingTop: 10,
  },
  todoContainer: {
    alignItems: 'center',
    marginTop: 10,
    marginBottom: 20,
  },
});
