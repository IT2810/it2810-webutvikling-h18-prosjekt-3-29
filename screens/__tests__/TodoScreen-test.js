import React from 'react';
import TodoScreen from '../TodoScreen';
import renderer from 'react-test-renderer';

//Checks that TodoScreen renders correctly
it('should render corrrectly', async () => {
  const tree = renderer.create(<TodoScreen/>).toJSON();
  expect (tree).toMatchSnapshot();
})
