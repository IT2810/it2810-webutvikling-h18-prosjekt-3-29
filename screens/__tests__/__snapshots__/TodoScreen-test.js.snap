// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render corrrectly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fcfcfc",
      "flex": 1,
    }
  }
/>
`;
